proxy/proxy.go

```diff
@@ -0,0 +1,23 @@
+package proxy
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+// Exec will invoke the given command and proxy any arguments for backwards compatibility.
+func Exec(command []string, args []string) error {
+	agent, err := exec.LookPath("circleci-agent")
+	if err != nil {
+		return errors.Wrap(err, "Could not find `picard` executable on $PATH; please ensure that build-agent is installed")
```

Comment:
I believe https://github.com/CircleCI-Public/circleci-cli/pull/55#discussion_r209290410 is still relevant. The text about "could not find x in path" will be duplicated because it's already in `err`.

Reply:
Whoops, that is correct.

The error was going to be:

```
Error: Could not find `picard` executable on $PATH; please ensure that build-agent is installed: exec: "picard": executable file not found in $PATH
```

Now I've changed it to be:

```
Error: please ensure that build-agent is installed: exec: "picard": executable file not found in $PATH
```